torch/testing/_internal/common_utils.py

```diff
@@ -1842,6 +1843,21 @@ def wrap_fn(self, *args, **kwargs):
         return wrap_fn
     return dec_fn
 
+def skipIfS390X(func=None, *, msg="test doesn't currently work on the s390x stack"):
```

Comment:
This looks fine.  You could also consider having `xfailIfS390X`, as xfail the test is usually a better mechanism than skipping it

Reply:
To elaborate a bit, `xfail` allows one to know if their change fixed something, while `skip` will leave the test in this state in perpetuity and likely cause author of the fix to write a new one, as they'll not be aware that skipped test existed